frontend/src/lib/components/UserLogin.svelte

```diff
@@ -1,102 +1,101 @@
 <script lang="ts">
-	import { goto } from '$app/navigation';
 	import { base } from '$app/paths';
 
 	import UserLoginUtil from '$lib/components//UserLoginUtil.svelte';
 	import AlertMessage from '$lib/components/AlertMessage.svelte';
 	import DataInput from '$lib/components/DataInput/DataInput.svelte';
-	import NavigationButtons from '$lib/components/Navigation/NavigationButtons.svelte';
 
-	import { hash, users, type UserData } from '$lib/stores/userStore';
-	import { Card, Heading } from 'flowbite-svelte';
-	import { onDestroy, onMount } from 'svelte';
+	import { goto } from '$app/navigation';
+	import { authCookieLogin, usersCurrentUser } from '$lib/client/services.gen';
+	import { type AuthCookieLoginData, type UserRead } from '$lib/client/types.gen';
+	import { currentUser } from '$lib/stores/userStore';
+	import { preventDefault } from '$lib/util';
+	import { Button, Card, Heading, Input } from 'flowbite-svelte';
+	import { _ } from 'svelte-i18n';
+
+	async function refresh(): Promise<string> {
+		const returned = await usersCurrentUser();
+		if (returned.error) {
+			console.log('Error getting current user: ', returned.error.detail);
+			currentUser.set(null as unknown as UserRead);
+			return returned.error.detail;
+		} else {
+			console.log('Successfully retrieved active user');
+			// TODO: remove this in the final version, this sets verified to true
+			// to emulate a successful user registration
+			returned.data.is_verified = true;
```

Comment:
I think it makes more sense to make the backend set is_verified=true on signup for now until #109 is implemented

Reply:
ok... did that now in `on_after_register` in the users.py file. find entry in database corresponding to the passed `user` object, set the verified flag -> write back to database.